src/main/java/Duke.java

```diff
@@ -1,10 +1,99 @@
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
+        welcomeMessage();
+
+        Scanner sc = new Scanner(System.in);
+        List<Task> tasks = new ArrayList<Task>();
+        Boolean exitProgram = false;
+
+        while (!exitProgram) {
+            String input = sc.nextLine();
+            String[] inputArray = input.split(" ", 2);
+
+            System.out.println("____________________________________________________________");
+
+            if (input.equals("list")) {
```

Comment:
Would it be better as a constant instead of a magic string?

Reply:
Yes, it would. Must have overlooked it thanks for pointing it out! :)